services/todo-legacy/repositories/todo.ts

```diff
@@ -0,0 +1,85 @@
+import {
+  AnyObject,
+  DataSourceConstructor,
+  DefaultCrudRepository,
+  Filter,
+  Model,
+  ModelBaseConstructor,
+  ObjectType,
+  Repository,
+  Where,
+  juggler
+} from '@loopback/repository';
+import { inject, Application } from '@loopback/core';
+import * as _ from 'lodash';
+export class TodoRepository {
+  // FIXME(kev): The juggler.ModelBase type doesn't carry any definitions
+  // for the CRUD functions that typically get mixed in when createModel
+  // makes a definition, so we have to use type "any" to avoid complaint!
+  // model: juggler.ModelBase | any;
+  model: juggler.PersistedModel | any;
```

Comment:
I see. @raymondfeng could you please look into this?

As far as I am concerned, we can land this version as long as we create a task/story to fix this ASAP after the pull request is landed. (I assume the fix would have to happen both in loopback-next and loopback-next-example repositories.)